tests/test_boostrap.py

```diff
@@ -52,5 +56,20 @@ def test_booststrap():
     # Bootstrap the system.
     subprocess.run([DUNE_EXE, "--bootstrap-system-full"], check=True)
```

Comment:
We are testing `--bootstrap-system-full`, but still need to test `--bootstrap-system`. Is it appropriate to do that here?